pilot/pkg/model/test/mockopenidserver.go

```diff
@@ -85,7 +85,7 @@ func (ms *MockOpenIDDiscoveryServer) Start() error {
 	}()
```

Comment:
Can we also log the error message returned from `server.ListenAndServe()`, this may probably explain why the server failed to start.

Reply:
I removed the log because of previous pre-check in racetest fail (there is some contention between log here and log in test code)

Reply:
Hmm, that sounds interesting, I didn't know the logging would somehow cause the racetest failure.
probably @geeknoid might know more about this?